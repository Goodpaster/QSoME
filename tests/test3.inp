#Geometries taken from https://pubs.acs.org/doi/suppl/10.1021/ja4102979/suppl_file/ja4102979_si_001.pdf
subsystem
Ne          0.0000        0.0000        1.000
hl_method_num 1
end

subsystem
Ne          0.0000        0.0000        0.000
end

basis
 default cc-pVDZ
end

env_method_settings
 env_method hf
 embed_settings
  cycles 50
  diis 2
 end
end

hl_method_settings
    hl_order 1
    hl_method rhf
end
opt_geom_settings
 cycles 100
end
