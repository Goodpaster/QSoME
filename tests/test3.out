iter:200:0               |ddm|:4.037526e-05               |Tr[DP]|:3.693136e-01
iter:200:1               |ddm|:1.593444e-07               |Tr[DP]|:3.693148e-01


                         Freeze-and-Thaw NOT converged
SUBSYS E
-1256.6841412149984
EMB E
11.330629320379558
PROJ E
0.3693136430492786
Subsystem 0 Energy:                                               -1244.98419825
SUBSYS E
-746.8745340780857
EMB E
-45.63471445349339
PROJ E
0.36931480279381196
Subsystem 1 Energy:                                                -496.22898082

********************************************************************************
TIMING: Freeze and Thaw                                               2747.3484s
KS-DFT  Energy:                                                   -1709.61811657
TIMING: Supersystem Energy                                               0.0000s
********************************************************************************
                             Env-in-Env Calculation
********************************************************************************
Env-in-Env Energy:                                                -1708.63147983
********************************************************************************
TIMING: Env. in Env. Energy                                              0.0030s
********************************************************************************
                           HL Subsystems Calculation
********************************************************************************
Overwritten attributes  energy_elec get_fock  of <class 'pyscf.scf.uhf.UHF'>
converged SCF energy = -1248.42766659924  <S^2> = 3.7536265  2S+1 = 4.0018128
